app/scenes/Settings/ImportExport.js

```diff
@@ -107,6 +264,11 @@ function ImportExport() {
     : false;
   const isImportable = hasCollections && hasDocuments;
 
+  const data = orderBy(exports.allData, sort, direction.toLowerCase()).slice(
```

Comment:
In the case of `sort = size` and `direction = asc`, the results are actually sorted in desc order. I have spent hours figuring out today, IDK what's wrong :/